styles.css

```diff
@@ -353,3 +342,238 @@ ul {
   padding: 12px 38px 12px 15px;
   margin-bottom: 22px;
 }
+
+/* Media querys */
+
+@media screen and (min-width: 768px) {
+  .headline-container {
+    background-image: url("images/Header-background-shapes-desktop.svg"), url("images/header-background-mobile.png");
+    background-repeat: no-repeat, no-repeat;
+    background-position: center, center;
+    background-size: contain, cover;
+    height: 93.5vh;
+  }
+
+  html {
+    font-size: 20px;
+  }
+
+  .hero-sections > * {
+    width: 65vw;
+  }
+
+  .btn-hamburger {
+    display: none;
+  }
+
+  .toolbar-menu-desktop {
+    display: flex;
+    align-items: center;
+    font-size: 0.8rem;
+    font-weight: 500;
+    color: #344563;
+    gap: 32px;
+  }
+
+  .toolbar {
+    position: fixed;
+    padding: 0 9%;
+    height: 6.5vh;
+    width: 100%;
+  }
+
+  /* About desktop */
+
+  .backgroud-about {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    width: 100%;
+    max-height: 700px;
+    padding: 0;
+    gap: 1%;
+  }
+
+  .about-sections {
+    width: 31%;
+    height: 44.5%;
+  }
+
+  .about-sections > * {
+    margin: 0;
+    padding: 0 0 0.5% 0;
+  }
+
+  .about-btn-container {
+    min-height: 72px;
+  }
+
+  .about-text {
+    font-size: 2rem;
+    line-height: 83%;
+    padding-bottom: 2%;
+  }
+
+  .about-paragraph,
+  .about-connect {
+    width: 100%;
+    font-size: 0.8rem;
+  }
+
+  .languages-flex {
+    justify-content: flex-start;
+    min-width: 47.4%;
+    height: 51.5%;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+  }
+
+  .active-dropdown {
+    width: 100%;
+    min-height: 100%;
+    padding-left: 4%;
+  }
+
+  .about-languages-container {
+    display: flex;
+    width: 414px;
+    min-height: 144px;
+  }
+
+  .dropdown-menu {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: flex-start;
+    min-width: 122px;
+    min-height: 120px;
+    background-color: #f7f7f9;
+    border-radius: 8px;
+    padding-left: 3%;
+    margin: 3% 0 0 3%;
+  }
+
+  /* Works desktop */
+
+  .card-order {
+    order: 1;
+  }
+
+  .cards {
+    padding: 114px 24px;
+    width: 100vw;
+    height: 2700px;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    padding: 16px;
+    margin: 0 146px;
+    gap: 12px;
+    width: 83%;
+    height: 496px;
+    background-color: #fff;
+    border: 1px solid #dfe1e6;
```

Comment:
- [ ] Based on project description, in order to lay out the elements on the page you should use `Grid` and `Flexbox`.
- Kindly refcator to your grid for the rending of the cards 👍 
- Check out [This resource](https://css-tricks.com/snippets/css/complete-guide-grid/) for reference 👍